src/Microsoft.ML.Core/EntryPoints/ComponentFactory.cs

```diff
@@ -37,6 +37,21 @@ public interface IComponentFactory<in TArg1, out TComponent> : IComponentFactory
         TComponent CreateComponent(IHostEnvironment env, TArg1 argument1);
     }
 
+    public class SimpleComponentFactory<TArg1, TComponent> : IComponentFactory<TArg1, TComponent>
```

Comment:
please add a summary comment to public class

Reply:
Done.

---
In reply to: [208311231](https://github.com/dotnet/machinelearning/pull/622#discussion_r208311231) [](ancestors = 208311231)